### PR TITLE
Feature/SU-9: Perfiles (roles): creación log histórico

### PR DESCRIPTION
**Pull request by fiter-julius-oketayot**
## Description
Jira Task:  https://fiterio.atlassian.net/browse/SU-9

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.jar
Comment (Napho) [inline gradle/wrapper/gradle-wrapper.jar:1]: This was intentionally removed by the suma team, if its added in error then add it to your personal gitignore

File: integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyActionIntegrationTests.java [hunk 1]
```diff
@@ -47,9 +47,9 @@
 import org.apache.fineract.client.models.PostLoansDelinquencyActionResponse;
 import org.apache.fineract.client.util.CallFailedRuntimeException;
 import org.apache.fineract.integrationtests.common.ClientHelper;
-import org.apache.fineract.integrationtests.common.loans.LoanTestLifecycleExtension;
 import org.apache.fineract.integrationtests.common.products.DelinquencyBucketsHelper;
 import org.apache.fineract.integrationtests.inlinecob.InlineLoanCOBHelper;
+import org.apache.fineract.integrationtests.sumas.loans.LoanTestLifecycleExtension;
```
Comment (Napho) [inline integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyActionIntegrationTests.java]: The package name change is bound to raise many conflicts if there is any upgrade from community, is this absolutely necessary?

File: fineract-provider/src/main/java/org/apache/fineract/useradministration/service/RoleWritePlatformServiceJpaRepositoryImpl.java [hunk 2]
```diff
@@ -103,22 +103,20 @@ private void handleDataIntegrityIssues(final JsonCommand command, final Throwabl
     public CommandProcessingResult updateRole(final Long roleId, final JsonCommand command) {
         try {
             this.context.authenticatedUser();
-
```
Comment (Napho) [inline fineract-provider/src/main/java/org/apache/fineract/useradministration/service/RoleWritePlatformServiceJpaRepositoryImpl.java:106]: Check that your computer is not formatting whitespaces..

**Review by Napho — CHANGES_REQUESTED** [review]

**Review by dstanca — APPROVED** [review]

File: fineract-provider/src/main/java/org/apache/fineract/useradministration/service/RoleWritePlatformServiceJpaRepositoryImpl.java (hunk 2 above)
Comment (fiter-julius-oketayot) [reply fineract-provider/src/main/java/org/apache/fineract/useradministration/service/RoleWritePlatformServiceJpaRepositoryImpl.java:106]: This was my update, I remove it intentionally

File: gradle/wrapper/gradle-wrapper.jar
Comment (fiter-julius-oketayot) [reply gradle/wrapper/gradle-wrapper.jar:1]: I noticed that it needed to run bootRun and integration tests
Let me try to find out if there is a way of downloading it before running the i-tests

Comment (Napho) [reply gradle/wrapper/gradle-wrapper.jar:1]: If you follow the earlier build script, you will notice that its downloaded  gradle before the build, maybe just move the download to a point before your tests can run.

File: integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyActionIntegrationTests.java (hunk 1 above)
Comment (fiter-julius-oketayot) [reply integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyActionIntegrationTests.java]: I needed to have a package to keep only only the tests specific the sumas
Allow me to review a better approach which would favor future upgrade. Perhaps I can create a separate

Comment (Napho) [reply integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyActionIntegrationTests.java]: A separate module might suit this better i think.

Comment (fiter-julius-oketayot) [reply integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyActionIntegrationTests.java]: Great, let me fix that 
thanks

File: gradle/wrapper/gradle-wrapper.jar
Comment (fiter-julius-oketayot) [reply gradle/wrapper/gradle-wrapper.jar:1]: Alright, that should then resolve it
Fixing it

**Pull request merged**